pkg/store/kotsstore/user_store.go

```diff
@@ -231,3 +232,64 @@ func (s *KOTSStore) flagSuccessfulLoginInDatabase() error {
 
 	return nil
 }
+
+// SetSharedPasswordBcrypt - set the shared password bcrypt hash in the kotsadm secret
+func (s *KOTSStore) SetSharedPasswordBcrypt(bcryptPassword []byte) error {
```

Comment:
is there not already a function for this, called by the CLI?

Can we use that instead?

Further - do we ever actually use a password from sqlite?

Reply:
I was bit hesitant to refactor existing code, added a new package `password` which is being used by cli as well as api